openai/cli.py

```diff
@@ -729,6 +724,17 @@ def help(args):
         help="A message in `{role} {content}` format. Use this argument multiple times to add multiple messages.",
         required=True,
     )
+
+    opt.add_argument(
+        "-e",
+        "--engine",
+        help="The engine to use. See https://platform.openai.com/docs/engines for more about what engines are available.",
```

Comment:
This link no longer seems to contain any documentation: https://platform.openai.com/docs/engines.

Could you update this to perhaps a Microsoft docs link and say something like "The engine to use when using Azure OpenAI Service. See [link here] for a list of engines available."

Reply:
change to https://learn.microsoft.com/en-us/azure/cognitive-services/openai/chatgpt-quickstart?pivots=programming-language-python